File: clang/game_fizz_buzz.py
"""
http://wiki.c2.com/?FizzBuzzTest
"""

for n in range(1, 101):
    line = ""
    if n%3 == 0:
        line = line + "Fizz"
    if n%5 == 0:
        line = line + "Buzz"

    if line:
        print line
    else:
        print n

File: clang/mess/game_fizz_buzz.rb
# http://wiki.c2.com/?FizzBuzzTest

result =
1.upto(100).map do |n|
  line = ""

  line += "Fizz" if n % 3 == 0
  line += "Buzz" if n % 5 == 0

  line.empty? ? n : line
end

puts result
